.gitignore

```diff
@@ -174,3 +174,13 @@ test-suite/.nra.cache
 
 plugins/ssr/ssrparser.ml
 plugins/ssr/ssrvernac.ml
+
+# dune files
```

Comment:
I would say "Dune auto-generated files" rather than "Dune files" for more clarity.

Reply:
Done, thanks! Note the updated PR description. Now focusing on the documentation.